test/client.ts

```diff
@@ -34,8 +34,12 @@ describe('Client', function () {
     assert.throws(() => new Client('wss//s:1'))
   })
 
-  it('Client connect() times out after 2 seconds', function () {
-    // TODO: Use a timer mock like https://jestjs.io/docs/en/timer-mocks
-    //       to test that connect() times out after 2 seconds.
+  it('Client connect() times out after 2 seconds', async function () {
+    const client = new Client('wss://ObviouslyNotARealDomain.com')
```

Comment:
I think this is supposed to be checking that it actually times out in 2 seconds and not some other interval.